src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1107,7 +1190,7 @@ private void visitTypeAsPrimary(JSType type) {
       }
     }
 
-    private void visitType(JSType typeToVisit) {
+    void visitType(JSType typeToVisit) {
```

Comment:
nope, it's a inner class, so I can access all methods. forgot my java (never knew it anyways).